tools/recetox_msfinder/macros.xml

```diff
@@ -2,7 +2,7 @@
     <token name="@TOOL_VERSION@">v3.5.2</token>
     <xml name="requirements">
         <requirements>
-            <container type="docker">recetox/recetox-msfinder:v3.5.2-rcx0</container>
+            <container type="docker">recetox/recetox-msfinder:v3.5.2-rcx1</container>
```

Comment:
Putting the docker image tag -rcx1 into the tool version was not legal if I remember it correctly?

Reply:
yes